UPGRADING.md

```
@@ -1,5 +1,165 @@
+- **Changed:** SSE reconnection wait bounded (#504)
+- **Changed:** Automatic pagination bounded in client (#505)
+- **Deprecated:** Roots and Sampling capabilities deprecated per SEP-2577 (#516)
+- **Fixed:** Exception messages no longer leaked to clients (#486)
+- **Fixed:** Invalid Params for unknown prompts and missing prompt arguments (#517)
+
+**1.1.0 (Aug 1, 2026):**
+- **Added:** 2026-07-28 as Latest Protocol Version (#476)
+- **Added:** Server tool annotations exposed on `MCP::Client::Tool` (#445)
+- **Fixed:** Explicit tool response content preserved (#469)
+
+### Migration steps
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)